### PR TITLE
New package: python-shapely

### PR DESCRIPTION
**Pull request by karl-nilsson**
also fixes the problem of cura dying on startup.

### CODE REVIEW

File: srcpkgs/python-shapely/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	vlicense LICENSE.txt LICENSE
+}
+
+python3-shapely_package() {
+
+	short_desc="${short_desc/Python2/Python3}"
+	pycompile_module="shapely"
+	make_depends="python3-devel python3-numpy geos-devel"
+	depends="python3-numpy geos"
+
```
Comment (maxice8) [inline srcpkgs/python-shapely/template]: extra line, remove

File: srcpkgs/python-shapely/template [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+homepage="https://github.com/Toblerity/Shapely"
+distfiles="https://github.com/Toblerity/Shapely/archive/${version}.tar.gz"
+checksum=d5163f46a99b96c4124b5df2a639dea2dcd7cb45b85128137807e3c9f06d3a16
+
+post_install() {
+	vlicense LICENSE.txt LICENSE
+}
+
+python3-shapely_package() {
+
```
Comment (maxice8) [inline srcpkgs/python-shapely/template]: extra line, remove

File: srcpkgs/python-shapely/template [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	short_desc="${short_desc/Python2/Python3}"
+	pycompile_module="shapely"
+	make_depends="python3-devel python3-numpy geos-devel"
```
Comment (maxice8) [inline srcpkgs/python-shapely/template]: There is no such thing as make_depends, and makedepends should only be set in the sourcepkg not any subpackage.

File: srcpkgs/python-shapely/template [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Template file for 'python-shapely'
+pkgname=python-shapely
+version=1.7a1
+revision=1
+wrksrc="Shapely-${version}"
+build_style=python-module
+pycompile_module="shapely"
+hostmakedepends="python-setuptools python3-setuptools python-Cython python3-Cython"
+makedepends="python-devel python-numpy geos-devel"
```
Comment (maxice8) [inline srcpkgs/python-shapely/template]: add the python3 subpackage make_depends here

File: srcpkgs/python-shapely/template [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+depends="python-numpy geos"
```
Comment (maxice8) [inline srcpkgs/python-shapely/template]: remove geos from depends=, it is found via shlibs.

**Comment by karl-nilsson** [comment]
@maxice8 fixed, thanks for the review!

File: srcpkgs/python3-shapely [hunk 6]
```diff
@@ -0,0 +1 @@
+python-shapely/
```
Comment (maxice8) [inline srcpkgs/python3-shapely]: no leading slash

```
cd srcpkgs
ln -s python-shapely python3-shapely
```

**Pull request merged**